Network/CMakeLists.txt

```diff
@@ -20,3 +52,5 @@ target_link_libraries(
     ${LIB}
     PRIVATE asio::asio
 )
+
+set_target_properties(${LIB} PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
What does it do ?

Reply:
It's to compile the network with .so files because it's a lib file it cause problem with FPIC.